# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../zlib/hotfix.patch
# Copyright (C) 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

                 from poppler-image.cpp:26:
/usr/include/zlib.h:1583:39: error: declaration of C function 'off_t gzseek64(gzFile, off_t, int)' conflicts with
/usr/include/zlib.h:1567:30: error: previous declaration 'off64_t gzseek64(gzFile, off64_t, int)' here
/usr/include/zlib.h:1584:39: error: declaration of C function 'off_t gztell64(gzFile)' conflicts with
/usr/include/zlib.h:1568:30: error: previous declaration 'off64_t gztell64(gzFile)' here
/usr/include/zlib.h:1585:41: error: declaration of C function 'off_t gzoffset64(gzFile)' conflicts with
/usr/include/zlib.h:1569:30: error: previous declaration 'off64_t gzoffset64(gzFile)' here
/usr/include/zlib.h:1586:46: error: declaration of C function 'uLong adler32_combine64(uLong, uLong, off_t)' conflicts with
/usr/include/zlib.h:1570:26: error: previous declaration 'uLong adler32_combine64(uLong, uLong, off64_t)' here
/usr/include/zlib.h:1587:44: error: declaration of C function 'uLong crc32_combine64(uLong, uLong, off_t)' conflicts with
/usr/include/zlib.h:1571:26: error: previous declaration 'uLong crc32_combine64(uLong, uLong, off64_t)' here

--- zlib-1.2.5/zlib.h
+++ zlib-1.2.5/zlib.h
@@ -1578,7 +1578,7 @@
 #  define gzoffset gzoffset64
 #  define adler32_combine adler32_combine64
 #  define crc32_combine crc32_combine64
-#  ifdef _LARGEFILE64_SOURCE
+#  ifndef _LARGEFILE64_SOURCE
      ZEXTERN gzFile ZEXPORT gzopen64 OF((const char *, const char *));
      ZEXTERN z_off_t ZEXPORT gzseek64 OF((gzFile, z_off_t, int));
      ZEXTERN z_off_t ZEXPORT gztell64 OF((gzFile));
